npm/design-system/src/components/FileExplorer/helpers/makeFileHierarchy.ts

```diff
@@ -0,0 +1,56 @@
+import { FileLike, Folder, FolderOrFile } from '../types'
+
+/**
+ * Split specs into group by their
+ * first level of folder hierarchy
+ *
+ * @param {Array<{name: string}>} specs
```

Comment:
It looks weird, because we have different type in the function declaration
```suggestion
```

Reply:
Yeah this was copy/paste foul.